packages/blocks/src/api/raw-handling/phrasing-content-reducer.js

```diff
@@ -8,18 +8,29 @@ import { includes } from 'lodash';
  */
 import { wrap, replaceTag } from '@wordpress/dom';
 
+const defaultColor = [ 'rgb(0, 0, 0)', 'rgba(0, 0, 0, 0)', 'black', '#000000' ];
+
 export default function phrasingContentReducer( node, doc ) {
 	// In jsdom-jscore, 'node.style' can be null.
 	// TODO: Explore fixing this by patching jsdom-jscore.
 	if ( node.nodeName === 'SPAN' && node.style ) {
 		const {
+			color,
 			fontWeight,
 			fontStyle,
 			textDecorationLine,
 			textDecoration,
-			verticalAlign,
+			verticgit statalAlign,
 		} = node.style;
 
+		// adds color to the wrapper
+		if ( color && ! defaultColor.includes( color ) ) {
+			const wrapper = doc.createElement( 'coloured' );
```

Comment:
My memory is hazy, but I remember using custom tags had surprising behaviors. I'm not completely opposed to custom tags as long as we can be sure they works as intended in all the browsers. Maybe using one of the allowed tags with a custom attribute like `data-has-color="true"` would be easier?

Reply:
Or maybe just checking for `style.color` later on would suffice?

Reply:
This 'custom tag' is not placed in html. I don't like the idea to put any custom tag directly to html. I use them only in the internal mechanics not to lose it, as every letter is stripped out of a span tag. Therefore, every custom tag is transformed back to span at the very end of all transformations

Reply:
We should not insert custom tags or data attributes. We should just preserve the `color` style property, and add the `has-inline-color` class.